lib/linalg/BasisGenerator.cpp

```diff
@@ -310,6 +324,78 @@ BasisGenerator::resetDt(
     }
 }
 
+void
+BasisGenerator::finalSummary(
```

Comment:
@axla-io I found there is a merge conflict in this PR, after your recently merged PR on nonlinear elasticity. It is due to you made some changes in the `BasisGeneratorFinalSummary` and I moved it to the `BasisGenerator` class. Is it safe to simply copy your changes here?

Reply:
Not sure if I understand the question and I'm a bit busy with a deadline. Can we talk about this on Thursday?

Reply:
Sure. No rush. Let's talk about it later.